xbmc/playlists/SmartPlayList.cpp

```diff
@@ -726,7 +726,7 @@ std::string CSmartPlaylistRule::FormatWhereClause(const std::string &negate, con
     table = "songview";
 
     if (m_field == FieldGenre)
-      query = negate + " EXISTS (SELECT 1 FROM song_genre, genre WHERE song_genre.idSong = " + GetField(FieldId, strType) + " AND song_genre.idGenre = genre.idGenre AND genre.strGenre" + parameter + ")";
+      query = negate + " EXISTS (SELECT 1 FROM song_genre, genre WHERE song_genre.idSong = " + GetField(FieldId, strType) + " AND song_genre.idGenre = genre.idGenre AND genre.name" + parameter + ")";
```

Comment:
These three changes to `CSmartPlaylist` are wrong because in the musicdb the column is still named `strGenre` and not `name` like in the videodb (see http://trac.kodi.tv/ticket/16104). Needs to be fixed and backported to Isengard.